src/components/resource/show.css

```diff
@@ -0,0 +1,3 @@
+.add-token-button {
+  margin-bottom: 0;
```

Comment:
Button has "marginBottom0" prop. I belive to be consistent with the codebase it is better to do it throught the prop